pt/santander.py

```diff
@@ -0,0 +1,362 @@
+                commercialProducts.remove('Consultas')
+            if 'Transferências' in commercialProducts:
+                commercialProducts.remove('Transferências')
+            if 'Carregamentos' in commercialProducts:
+                d["prepaid_top_up:mobile"] = "yes"
+                commercialProducts.remove('Carregamentos')
+            if 'Pagamentos de serviços' in commercialProducts:
+                commercialProducts.remove('Pagamentos de serviços')
+            if 'Requisição de cheques' in commercialProducts:
+                commercialProducts.remove('Requisição de cheques')
+            if 'DEPÓSITOS E CHEQUES' in commercialProducts:
+                d["cash_in"] = "yes"
+                d["cheque_in"] = "yes"
+                commercialProducts.remove('DEPÓSITOS E CHEQUES')
+            if 'LEVANTAMENTOS E DEPÓSITOS' in commercialProducts:
+                d["cash_out"] = "yes"
+                d["cash_in"] = "yes"
+                commercialProducts.remove('LEVANTAMENTOS E DEPÓSITOS')
+            if 'LEVANTAMENTOS E CHEQUES' in commercialProducts:
+                d["cash_out"] = "yes"
+                d["cheque_in"] = "yes"
+                commercialProducts.remove('LEVANTAMENTOS E CHEQUES')
+            if 'SERVIÇO DE BALCÃO' in commercialProducts:
+                commercialProducts.remove('SERVIÇO DE BALCÃO')
+            if 'MÁQUINA DE DEPÓSITOS' in commercialProducts:
+                d["cash_in"] = "yes"
+                commercialProducts.remove('MÁQUINA DE DEPÓSITOS')
+            if 'DEPÓSITO DE NOTAS NA CONTA DE TERCEIROS' in commercialProducts:
+                d["cash_in"] = "yes"
+                commercialProducts.remove('DEPÓSITO DE NOTAS NA CONTA DE TERCEIROS')
+            if 'NÃO TEM SERVIÇO DE CAIXA' in commercialProducts:
+                commercialProducts.remove('NÃO TEM SERVIÇO DE CAIXA')
+            if 'APENAS SERVIÇOS COMERCIAIS DISPONÍVEIS' in commercialProducts:
+                commercialProducts.remove('APENAS SERVIÇOS COMERCIAIS DISPONÍVEIS')
+
+            assert commercialProducts == set()
+        
+        if "spokenlanguages" in nd:
+            for language in nd["spokenlanguages"]:
+                d[f"language:{language.lower()}"] = "yes"
+
+        if "dialogAttribute" in nd:
+            if "WIFI" in nd["dialogAttribute"]:
+                d["internet_access"] = "wlan"
+            if "RETIRO_CON_CODIGO" in nd["dialogAttribute"]:
+                d["cash_out"] = "yes"
+            if "MULTICAJERO" in nd["dialogAttribute"]:
+                d["atm"] = "yes"
+                d["cash_in"] = "yes"
+
+        if d.kind == "new" and not d["addr:street"] and not (d["addr:housenumber"] or d["nohousenumber"] or d["addr:housename"]):
+            d["x-dld-addr"] = nd["location"]["address"]
+
+    for d in old_data:
+        if d.kind != "old":
+            continue
+        ref = d[REF]
+        if ref and any(nd for nd in new_data if ref == nd.get("poicode")):
```

Comment:
Since the logic above implies there could be multiple values in `ref`, need to avoid marking the good POIs for removal here:
```suggestion
        ref = d[REF].split(";") if d[REF] else None
        if ref and any(nd for nd in new_data if nd["poicode"] in ref):
```